docs/sources/mimir/configure/configure-ingester-circuit-breakers.md

```diff
@@ -0,0 +1,83 @@
+---
+aliases:
+  - ../configuring/configuring-ingester-circuit-breakers/
+description: Learn how to configure ingester's circuit breakers.
+menuTitle: Ingester's circuit breakers
+title: Configure Grafana Mimir Ingester circuit breakers
+---
+
+# Configure Grafana Mimir Ingester circuit breakers
+
+## How do Grafana Mimir Ingester circuit breakers work?
```

Comment:
perhaps a section on why it would be a good idea to enable circuit breakers and when not to can help. That way people don't use it for things it wasn't designed and it (ideally) shows up as a search result when you google your Mimir problem

Reply:
Hi @dimitarvdimitrov, @tacole02 
I have added a background section, where I explained more details about the circuit breaker pattern.
Moreover, I explicitly stated what circuit breakers protect the ingesters from.
Could you please have a look and let me know what you think?